fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -844,126 +840,6 @@ void repartitionAndMerge(Index index, Tuple groupingKey, int repartitionCount, i
         }
     }
 
-    static Stream<Arguments> randomizedRepartitionTest() {
```

Comment:
This test was moved to `LuceneIndexMaintenanceTest`. The primary reason I moved it was to decouple it from the metadata used in this class. The metadata in this class has a bunch of other, unused types, that don't have a `group` field, thus preventing `deleteRecordsWhere`